projects/storefrontlib/src/cms-components/content/banner/banner.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-container *ngIf="component.data$ | async as data">
   <cx-generic-link
     [url]="data.urlLink"
```

Comment:
now I'm wondering since banner component supports external, should we include [attr.class]=class to support `styleclasses` from the BannerComponent CMS property? (May be out of scope)

What do you think?

Reply:
done ;)